package.json

```diff
@@ -31,6 +31,7 @@
     "view:reports": "open reports/coverage/lcov-report/index.html"
   },
   "dependencies": {
+    "@ltipton/args-parse": "^1.0.0",
     "@ltipton/jsutils": "^5.0.0",
```

Comment:
Yah, we should.
It's a repo I manage, so technically if i don't follow semver, then it's my fault. The others are locked because we can't ensure they follow semver properly. expo-cli is a perfect example of that, but to keep it consistent We should lock it down. 
I'll update it to do that